src/index.ts

```diff
@@ -46,9 +47,7 @@ void (async function () {
   let bot: DraupnirBotModeToggle | null = null;
   let client: MatrixClient;
   try {
-    const storagePath = path.isAbsolute(config.dataPath)
-      ? config.dataPath
-      : path.join(__dirname, "../", config.dataPath);
+    const storagePath = await realpath(config.dataPath);
```

Comment:
Hmm, how come you use realPath here but then not in `SqliteRoomStateBackingStore.create`?

Reply:
Has it changed the behavior?